lib/KDropdownMenu.vue

```diff
@@ -34,6 +35,11 @@
       UiMenu,
     },
     props: {
+       // Override default prop value
```

Comment:
Can we replace this comment with documentation instead

```javascript
/**
 * The dropdown menu popover flips its position to avoid overflows within the parent. Setting it to false disables the flipping behavior.
 */
```

@MisRob, happy to here your thought if this developer friendly language :)

Reply:
That'd be helpful documentation definitely, thank you

Reply:
Thank you for the feedback @akolson. I have made the requested changes in the latest commit.

Reply:
Does this mean I have to submit another PR showcasing the override for the template in Kolibiri rendering the KDropdownMenu component?

Reply:
Yes, I think we could start by overriding the value in Kolibri: _Facility Settings_, where the bug was initially caught. Please see https://github.com/learningequality/kolibri-design-system/issues/432 for specifics. You can test your KDS changes locally in kolibri by running 
```
yarn run devserver-with-kds /path/to/your/local/kolibri-design-system
```
The rest of the overrides will be done as and when its needed.